### ISSUE TITLE
Macro Icon being a ?

### ISSUE THREAD

**Issue by bolte-io** [brittyazel/Neuron#316]
Unsure if this is just this addon in combination with Super Duper Macro.
However, if I create a macro to toggle my bags as a floating macro (accessed by /click sdf_NameOfMacro)

Then enter right click mode in Neuron, and set an icon in there, with the code being /click sdf_Bags (in my case) the icon shows on the Neuron bar, but with a ? icon and unknown as the title.

This would imply that the icon selection is not working in Neuron, the title I would expect not to be shown as it is a floating macro and is code, not a spell being cast, but the icon should show if I have selected one I would imagine.

The bar: 
![image](https://user-images.githubusercontent.com/51535687/65537668-8fabbd80-defd-11e9-9b14-05c53d03817f.png)

The macro screen: 
![image](https://user-images.githubusercontent.com/51535687/65537704-a18d6080-defd-11e9-9491-9c5cc2351400.png)

The code actually works as expected and opens my bags.
Shouldn't the icon be selected as I chose?

**Comment by brittyazel** [comment]
Yes the icon selector has been broken for a little while. I'll see if I can get that sorted for you

**Comment by brittyazel** [comment]
Is this still broken with 1.3.2?

**Comment by bolte-io** [comment]
I will need to revisit this, I have changed my UI again since, I will let you know when I get the time, If nobody else reproduces before me.
